src/test/java/it/gov/pagopa/payhub/activities/service/paymentsreporting/PaymentsReportingMapperServiceTest.java

```diff
@@ -0,0 +1,96 @@
+
+		ctFlussoRiversamento.setDataRegolamento(toXMLGregorianCalendar(gregorianCalendar));
+		ctFlussoRiversamento.setDataOraFlusso(toXMLGregorianCalendar(gregorianCalendar));
+
+		CtIstitutoMittente istitutoMittente = new CtIstitutoMittente();
+		istitutoMittente.setDenominazioneMittente("PSP Mittente");
+		CtIdentificativoUnivoco identificativoMittente = new CtIdentificativoUnivoco();
+		identificativoMittente.setCodiceIdentificativoUnivoco("12345");
+		identificativoMittente.setTipoIdentificativoUnivoco(B);
+		istitutoMittente.setIdentificativoUnivocoMittente(identificativoMittente);
+		ctFlussoRiversamento.setIstitutoMittente(istitutoMittente);
+
+		CtIstitutoRicevente istitutoRicevente = new CtIstitutoRicevente();
+		istitutoRicevente.setDenominazioneRicevente("Org Ricevente");
+		CtIdentificativoUnivocoPersonaG identificativoRicevente = new CtIdentificativoUnivocoPersonaG();
+		identificativoRicevente.setCodiceIdentificativoUnivoco("54321");
+		identificativoRicevente.setTipoIdentificativoUnivoco(G);
+		istitutoRicevente.setIdentificativoUnivocoRicevente(identificativoRicevente);
+		ctFlussoRiversamento.setIstitutoRicevente(istitutoRicevente);
+
+		IngestionFlowFileDTO ingestionFlowFileDTO = new IngestionFlowFileDTO();
+		ingestionFlowFileDTO.setOrg(OrganizationDTO.builder().orgId(1L).build());
+		ingestionFlowFileDTO.setIngestionFlowFileId(1L);
+		ingestionFlowFileDTO.setCreationDate(TestUtils.DATE);
+
+		CtDatiSingoliPagamenti singlePayment = new CtDatiSingoliPagamenti();
+		singlePayment.setIdentificativoUnivocoVersamento("vers123");
+		singlePayment.setIdentificativoUnivocoRiscossione("ris123");
+		singlePayment.setIndiceDatiSingoloPagamento(1);
+		singlePayment.setSingoloImportoPagato(BigDecimal.valueOf(200.00D));
+		singlePayment.setCodiceEsitoSingoloPagamento("OK");
+		singlePayment.setDataEsitoSingoloPagamento(toXMLGregorianCalendar(gregorianCalendar));
+
+		ctFlussoRiversamento.getDatiSingoliPagamenti().addAll(List.of(singlePayment));
+
+		// When
+		List<PaymentsReportingDTO> result = mapper.mapToDtoList(ctFlussoRiversamento, ingestionFlowFileDTO);
+
+		PaymentsReportingDTO firstDTO = result.get(0);
+		// Then
+		assertEquals(1, result.size());
+		assertEquals("flow123", firstDTO.getFlowIdentifierCode());
+		assertEquals("PSP Mittente", firstDTO.getSenderPspName());
+		assertEquals("Org Ricevente", firstDTO.getReceiverOrganizationName());
+		assertEquals(1L, firstDTO.getTotalPayments());
+		assertEquals(100_050L, firstDTO.getTotalAmountCents());
+		assertEquals("vers123", firstDTO.getCreditorReferenceId());
+		assertEquals("ris123", firstDTO.getRegulationId());
+		assertEquals(1, firstDTO.getTransferIndex());
+		assertEquals(20_000L, firstDTO.getAmountPaidCents());
+		assertEquals("OK", firstDTO.getPaymentOutcomeCode());
+		assertNotNull(firstDTO.getPayDate());
+
+		TestUtils.checkNotNullFields(firstDTO, "creationDate", "lastChangeDate", "paymentsReportingId");
```

Comment:
ok for the id null, but creationDate and lastChangeDate should be null?

Reply:
removed, there's no default value